src/ui/components/VPNLinkButton.qml

```diff
@@ -69,14 +64,24 @@ RoundButton {
         }
     ]
 
-    VPNMouseArea {
+    VPNUIStates {
+        colorScheme: root.baseColor
+        itemToFocus: root
 
+        VPNFocusBorder {
+            border.color: root.linkColor.focusBorder
```

Comment:
Tiniest of nits and doesn't need to block but let's change this to `border.color: root.linkColor.defaultColor` so that the red border around the sign out links matches the red border around the red trashbin icon buttons in the devices view.